qiskit/visualization/state_visualization.py

```diff
@@ -630,7 +630,7 @@ def phase_to_rgb(complex_number):
     return rgb
 
 
-def plot_state_qsphere(rho, figsize=None, ax=None):
+def plot_state_qsphere(rho, figsize=None, ax=None, show_state_labels=True, show_state_phases=True):
```

Comment:
Could we add an argument that says whether we are using degrees or radians? Perhaps `use_degrees`.  I feel like people might have a preference so it would be best to give them the option.

Reply:
This is a good idea. I added a `use_degrees` argument in the follow-up commit